test/NuGet.Tests.Apex/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup Condition="'$(_WasCommonPropsImported)' != 'true'">
+    <TestProject Condition="'$(TestProject)' == ''">true</TestProject>
```

Comment:
Only two of the projects under this directory are test projects. The others are class libraries used by one of the test projects. Although, I assume that this is maintaining the existing behaviour.

Reply:
Yeah I'm doing what's currently configured which is that all of the projects in here import `common.test.props`:

```
D:\nuget3\test\NuGet.Tests.Apex>findstr /spc:"common.test.props" *
NuGet.Console.TestContract\NuGet.Console.TestContract.csproj:  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'NuGet.sln'))\build\common.test.props" />
NuGet.OptProf\NuGet.OptProf.csproj:  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'NuGet.sln'))\build\common.test.props" />
NuGet.PackageManagement.UI.TestContract\NuGet.PackageManagement.UI.TestContract.csproj:  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'NuGet.sln'))\build\common.test.props" />
NuGet.Tests.Apex\NuGet.Tests.Apex.csproj:  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'NuGet.sln'))\build\common.test.props" />

D:\nuget3\test\NuGet.Tests.Apex>dir *.csproj /s /b
NuGet.Console.TestContract\NuGet.Console.TestContract.csproj
NuGet.OptProf\NuGet.OptProf.csproj
NuGet.PackageManagement.UI.TestContract\NuGet.PackageManagement.UI.TestContract.csproj
NuGet.Tests.Apex\NuGet.Tests.Apex.csproj
```

Do you think I should change this?

Reply:
For the sake of getting a "big" change in sooner rather than later, plus it already works like this, I think it's fine to keep it as is. Just may be worth something revisiting in the future. Having said that, I don't know how `<TestProject>true` changes the build, so I don't know how important it is (but again, it hasn't broken anything so far).